clang/test/Misc/pragma-attribute-strict-subjects.c

```diff
@@ -90,7 +90,7 @@ struct testSubset1Struct { };
 
 #pragma clang attribute pop
 
-#pragma clang attribute push (__attribute__((abi_tag("a"))), apply_to = variable)
+#pragma clang attribute push (__attribute__((abi_tag("a"))), apply_to = variable) // expected-warning {{attribute 'abi_tag' cannot be applied to a 'void' parameter}}
```

Comment:
```suggestion
#pragma clang attribute push (__attribute__((abi_tag("a"))), apply_to = variable)
// expected-warning@-1 {{attribute 'abi_tag' cannot be applied to a 'void' parameter}}
```
Same changes elsewhere; just keeps the line length more reasonable when reading the tests.